### PR TITLE
zh-CN: create `extension.onRequestExternal`

### PR DESCRIPTION
**Pull request by PassionPenguin**

### CODE REVIEW

File: files/zh-cn/mozilla/add-ons/webextensions/api/extension/onrequestexternal/index.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  sendResponse, // 函数
+) { })
+chrome.extension.onRequestExternal.removeListener(listener)
+chrome.extension.onRequestExternal.hasListener(listener)
+```
+
+事件有三个函数：
+
+- `addListener(listener)`
+  - : 添加一个监听该事件的监听器。
```
Comment (yin1999) [inline files/zh-cn/mozilla/add-ons/webextensions/api/extension/onrequestexternal/index.md:30]:
```suggestion
  - : 添加一个该事件的监听器。
```

File: files/zh-cn/mozilla/add-ons/webextensions/api/extension/onrequestexternal/index.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+title: extension.onRequestExternal
+slug: Mozilla/Add-ons/WebExtensions/API/extension/onRequestExternal
+l10n:
+  sourceCommit: b8a0743ca8b1e1b1b1a95cc93a4413c020f11262
+---
+
+{{AddonSidebar}}
+
+> [!WARNING]
+> 此 API 在 Firefox 中未实现，因为自 Chrome 33 起已被弃用。请改用 [runtime.onMessageExternal](/zh-CN/docs/Mozilla/Add-ons/WebExtensions/API/runtime/onMessageExternal)。
```
Comment (yin1999) [inline files/zh-cn/mozilla/add-ons/webextensions/api/extension/onrequestexternal/index.md:11]: 两个文档的句式麻烦统一一下：#23540

Comment (PassionPenguin) [reply files/zh-cn/mozilla/add-ons/webextensions/api/extension/onrequestexternal/index.md:11]:
```suggestion
> 因为该方法自 Chrome 33 起被弃用，该 API 在 Firefox 中并未被实现。请改用 [runtime.onMessageExternal](/zh-CN/docs/Mozilla/Add-ons/WebExtensions/API/runtime/onMessageExternal)。
```

**Review by yin1999 — APPROVED** [review]
:+1:

**Pull request merged**